src/Microsoft.TemplateEngine.Core/Util/ProcessorState.cs

```diff
@@ -345,7 +347,7 @@ public bool Run()
                 _target.Write(CurrentBuffer, CurrentBufferLength - toWrite, toWrite);
             }
 
-            _target.Flush();
+            _target.FlushToTarget();
```

Comment:
`Flush` also happens at line 249

Reply:
For now I'm keeping the pre-existing logic (configurable flushing after specific # of bytes send to Processor)